data-raw/qc-markdowns/seine-snorkel-data/feather-river/combined_feather_snorkel_qc.Rmd

```diff
@@ -0,0 +1,143 @@
+
+### Pre-2004
+
+#### Columns Removed
+
+- `flow` and `adj_flow` - sparsely recorded and values not on the same scale as `river_flow`
+- `fish_depth` seemed redundant with `river_depth`
+- `max_fork_length` seemed redundant with `fork_length`
+
+columns created/renamed:
+- `river_flow` renamed to `velocity`
+- `river_depth` renamed to `water_depth_m`
+- `huc_unit` renamed to `unit_type`
+
+```{r}
+clean_snorkel_pre_2004 <- snorkel_pre_2004 |> 
+  select(-c(flow, adj_flow, fish_depth, max_fork_length)) |> 
+  rename(velociy = river_flow,
+         water_depth_m = river_depth,
+         unit_type = huc_unit) |> 
+  glimpse()
+```
+
+### Post-2004
+
+#### Columns Removed
+
+- `survey_comments` and `observation_comments` - not necessary for final dataset
+- `time_of_temperature`, `observation_id`, `lwd_number` - not necessary for final dataset
+
+columns created/renamed:
+- `flow` renamed to `velocity`
+- `weather_code` renamed to `weather`
+- `hydrology_code` renamed to `hydrology`
+- 
+
+```{r}
+# get lookup tables from pre-2004 table for some codes
+db_filepath <- here::here("data-raw", "qc-markdowns", "seine-snorkel-data", "feather-river", "feather-river-db.mdb")
+library(Hmisc)
+lookup_HUC_cover <- mdb.get(db_filepath, "HUCcoverLU")
+lookup_HUC_o_cover <- mdb.get(db_filepath, "HUCOcoverLU")
+lookup_HUC_substrate <- mdb.get(db_filepath, "HUCsubstrateLU")
+detach(package:Hmisc)
+
+# TODO still need to do instream cover codes and substrate - they are complicated and time consuming
+
+# first join the lookup tables
+clean_snorkel_post_2004_lu <- snorkel_post_2004 |> 
+  left_join(lookup_HUC_o_cover, by = c("overhead_cover" = "CoverCode")) |> 
+    mutate(Cover = case_when(overhead_cover == 12 ~ "Overhead Object/Veg. 0 - 0.5m, Overhead Object/Veg. 0.5 - 2m",
+                           overhead_cover == 13 ~ "Overhead Object/Veg. 0 - 0.5m, Submerged Aquatic Veg/Algae",
+                           overhead_cover == 4 ~ "unknown (code 4)",
+                           TRUE ~ Cover),
+           Cover = str_to_lower(Cover)) |> 
+  select(-c(overhead_cover)) |> 
+  rename(overhead_cover = Cover) |> 
+  glimpse()
+
+clean_snorkel_post_2004 <- clean_snorkel_post_2004_lu |> 
+  select(-c(survey_comments, observation_comments, time_of_temperature,
+            observation_id, lwd_number)) |> 
+  rename(velocity = flow,
+         weather = weather_code,
+         hydrology = hydrology_code) |> 
+  mutate(hydrology = str_to_lower(hydrology),
+         substrate = as.character(substrate)) |> 
+  glimpse()
+```
+
+
+## Combine data:
+
+```{r}
+all_snorkel_data <- bind_rows(clean_snorkel_post_2004, 
+                           clean_snorkel_pre_2004)  |> glimpse()
+```
+
+
+## Save Cleaned Data to Google Cloud
+
+```{r}
+knitr::kable(all_snorkel_data  |> head())
+```
+
+```{r, eval=FALSE}
+#write_csv(all_snorkel_data, #here::here("data","redd-carcass-holding","standard_all_redd_data.csv"))
```

Comment:
Looks like this is still left over from a copied md. Please update and save combined dataset to cloud